TestProjects/com.unity.mobile-notifications-sample/Assets/Scripts/ScriptableObjects/AndroidNotificationTemplate.cs

```diff
@@ -36,6 +36,7 @@ public class AndroidNotificationTemplate : ScriptableObject
         [TextArea] public string IntentData = "";
         public bool ShowTimestamp = false;
         public long RepeatInterval;
+        public bool ShowInForeground = true;
```

Comment:
Is technically possible to get notification to show up both while app is in background or foreground. Wondering if this shouldn't be an enum with bit wise values?

Reply:
Notifications in general are designed to be shown while app is in background in order to bring user back to app. But it's possible send them while app is in foreground too.
So ShowInForeground means send notification disregarding foreground/background, because you can safely assume notification should be sent while app is in background.